### PR TITLE
Fix the build

### PR DESCRIPTION
**Pull request by cedricziel**

### CODE REVIEW

**Comment by cedricziel** [comment]
@NamelessCoder could you please proof-read? Thx.

**Comment by NamelessCoder** [comment]
As soon as time allows!

**Comment by cedricziel** [comment]
No rush ;)

File: Classes/ViewHelpers/Extension/Path/SiteRelativeViewHelper.php [hunk 1]
```diff
@@ -30,6 +31,10 @@ class SiteRelativeViewHelper extends AbstractExtensionViewHelper
      */
     public static function renderStatic(array $arguments, \Closure $renderChildrenClosure, RenderingContextInterface $renderingContext)
     {
+        if (true === version_compare('9.0', TYPO3_branch, '<=')) {
```
Comment (NamelessCoder) [inline Classes/ViewHelpers/Extension/Path/SiteRelativeViewHelper.php]: Is this right? Looks like this means that on 9.0+ we'll then be calling the deprecated, possibly removed, method below? Should that not instead perhaps target 7.6 specifically and call the deprecated method only on that version?

**Comment by peterkraume** [comment]
I think this patch blocks the Travis checks in my PR #1425.
Would be great to get this one sorted out soon!

**Comment by cedricziel** [comment]
Should be fine now for the purpose of the PR itself, but may not serve @peterkraume well, since TYPO3 master did lose render methods in the meantime ^^

**Comment by peterkraume** [comment]
So what do I need to do to get #1425 merged?

**Pull request merged**